rsmtool/notebooks/explanations/shap_values.ipynb

```diff
@@ -152,6 +153,39 @@
     "display(bottom_5_table)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "mean_values = df_abs_shap[\"abs. mean shap\"]\n",
+    "rows_with_zero_mean_values = df_abs_shap[df_abs_shap[\"abs. mean shap\"] == 0]\n",
+    "\n",
+    "msg = (\"#### Features with zero mean Shap value\\n The features in table below \"\n",
```

Comment:
The features in table below ->  The features in **the** table below
(for some reason I don't have the option to edit in the text)